driver/src/main/java/org/neo4j/driver/v1/Result.java

```diff
@@ -103,4 +112,7 @@
      * @return a summary for the whole query
      */
     ResultSummary summarize();
+
+    @Override
+    void close();
```

Comment:
Would be nice to document that implementations might throw if `Result` is already closed.